app.rb

```diff
@@ -22,7 +22,20 @@ def init
 def handle_event(event:, context:)
   init
 
-  return handle_is_research(event)
+  path = event["path"]
+  method = event["httpMethod"].downcase
+
+  if method == 'get' && path == "/docs/is-research"
+    return handle_swagger
+  else
+    return handle_is_research(event)
```

Comment:
should probably just `respond 400, message: "Bad method"` if method isn't 'get'.. And thereafter switch on the path (`case` stmt could work well here). As it is, you'll call `handle_is_research` if the lambda is wired up to handle a POST to `/docs/is-research`